README.md

```
@@ -30,38 +22,36 @@
+  </h3>
 
-<div align="center">
-  <a href="https://go.copilotkit.ai/gh-linkedin" target="_blank">
-    <img src="https://github.com/user-attachments/assets/e33e7ebb-f5fc-4775-81b0-d5dd6865271a" alt="LI">
-  </a>
-  <a href="https://go.copilotkit.ai/gh-twitter" target="_blank">
-    <img src="https://github.com/user-attachments/assets/14e57c97-70ac-4f9a-88f5-67028107794f" alt="Discord">
-  </a>
+  <div align="center" style="display: flex; justify-content: center; gap: 16px; height: 43px; margin: 0;">
+    <a href="https://trendshift.io/repositories/5730" target="_blank">
+      <img src='./assets/buttons/linkedin.svg'/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)